pkg/identity/identity.go

```diff
@@ -76,3 +79,87 @@ func (id *signingIdentity) Credentials() []byte {
 func (id *signingIdentity) Sign(message []byte) ([]byte, error) {
 	return id.sign(message)
 }
+
+func CreateSigner(privateKeyPath, certificatePath, mspID string) (SigningIdentity, error) {
+	priv, err := GetPrivateKey(privateKeyPath)
+	if err != nil {
+		return nil, err
+	}
+
+	cert, _, err := GetCertificate(certificatePath)
+	if err != nil {
+		return nil, err
+	}
+
+	return NewPrivateKeySigningIdentity(mspID, cert, priv)
+}
+
+func GetPrivateKey(f string) (*ecdsa.PrivateKey, error) {
+	in, err := os.ReadFile(f)
+	if err != nil {
+		return nil, err
+	}
+
+	k, err := PEMtoPrivateKey(in)
+	if err != nil {
+		return nil, err
+	}
+
+	key, ok := k.(*ecdsa.PrivateKey)
+	if !ok {
+		return nil, errors.New("expecting ecdsa key")
+	}
+
+	return key, nil
+}
+
+func GetCertificate(f string) (*x509.Certificate, []byte, error) {
+	in, err := os.ReadFile(f)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	block, _ := pem.Decode(in)
+
+	c, err := x509.ParseCertificate(block.Bytes)
+	return c, in, err
+}
+
+// PEMtoPrivateKey unmarshals a pem to private key
+func PEMtoPrivateKey(raw []byte) (interface{}, error) {
```

Comment:
I'm not sure a lower-case **t** in for **to** fits Go naming conventions, and doesn't match **DERToPrivateKey**. If you want to avoid all the capitals in **PEMTo** then you could name it `PrivateKeyFromPEM` instead

Reply:
move to internal function